scripts/emojicheck.js

```diff
@@ -1,5 +1,8 @@
 // @ts-check
 
+const https = require("https");
```

Comment:
we use axios in scripts/dictionary — probably cleaner to use it here, too

Reply:
Nice. I didn't want to add a dependency, but axios is a lot cleaner.